content/en/docs/tasks/administer-cluster/encrypt-data.md

```diff
@@ -180,63 +281,108 @@ program to retrieve the contents of your Secret.
+an update on Secret will encrypt that content at rest.
 
-Since Secrets are encrypted on write, performing an update on a Secret will encrypt that content.
+You can make this change across all Secrets in your cluster:
 
 ```shell
+# Run this as an administrator that can read and write all Secrets
 kubectl get secrets --all-namespaces -o json | kubectl replace -f -
 ```
 
-The command above reads all Secrets and then updates them to apply server side encryption.
+The command above reads all Secrets and then updates them with the same data, in order to
+apply server side encryption.
 
 {{< note >}}
 If an error occurs due to a conflicting write, retry the command.
-For larger clusters, you may wish to subdivide the secrets by namespace or script an update.
+It is safe to run that command more than once.
+
+For larger clusters, you may wish to subdivide the Secrets by namespace,
+or script an update.
 {{< /note >}}
 
+## Cleanup {#cleanup-all-secrets-encrypted}
+
+You **must** ensure that all existing Secrets are encrypted at rest before
+performing this cleanup step, because this change prevents the API server from
+retrieving Secret resources (API path `secrets`) that are not encrypted.
+
+Once all Secrets in your cluster are encrypted, you can remove the `identity`
+part of the encryption configuration. For example:
+
+```yaml
+---
+apiVersion: apiserver.config.k8s.io/v1
+kind: EncryptionConfiguration
+resources:
+  - resources:
+      - secrets
+    providers:
+      - aescbc:
+          keys:
+            - name: key1
+              secret: <BASE 64 ENCODED SECRET>
+      - identity: {} # REMOVE THIS LINE
+```
+
+…and then restart each API server in turn. This change prevents the API server
+from accessing a plain-text Secret, even by accident.
+
 ## Rotating a decryption key
 
 Changing a Secret without incurring downtime requires a multi-step operation, especially in
 the presence of a highly-available deployment where multiple `kube-apiserver` processes are running.
 
-1. Generate a new key and add it as the second key entry for the current provider on all servers
-1. Restart all `kube-apiserver` processes to ensure each server can decrypt using the new key
-1. Make the new key the first entry in the `keys` array so that it is used for encryption in the config
-1. Restart all `kube-apiserver` processes to ensure each server now encrypts using the new key
-1. Run `kubectl get secrets --all-namespaces -o json | kubectl replace -f -` to encrypt all
-   existing Secrets with the new key
-1. Remove the old decryption key from the config after you have backed up etcd with the new key in use
-   and updated all Secrets
-
-When running a single `kube-apiserver` instance, step 2 may be skipped.
+1. Generate a new key and add it as the second key entry for the current provider on all
+   control plane hosts (strictly speaking: on all hosts that run a `kube-apiserver`).
```

Comment:
```suggestion
   nodes that run a `kube-apiserver` process.
```

Reply:
My wording avoids assuming that the `kube-apiserver` is running inside Pods. I will try to find another wording that also works for the common case where there are control plane nodes.